### PR TITLE
173472187 add query download link

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR adds a link to download the query results CSV file when the user presses the Generate CSV Link button.  The link is valid for 10 minutes.  Changes include:
- add `amazon-s3-uri` for parsing bucket, region, and key info from output location URI
- when button is pressed get info from output location URI and use info to make signed link
- show error messages if steps fail
- CSS tweak to better display app
- remove obsolete files

To test, you need to login to Athena and set the workgroup to the one that you created based on the info in the example app.  You then need to ensure that your query results are being stored in the `workgroup-output` folder in the `concordqa-report-data` bucket (not `s3://concordqa-report-data/athena-results/`).  Under settings, change the query result location to be: 
`s3://concordqa-report-data/workgroup-output/[WORKGROUP-NAME]`.  For example, in the workgroup below, I set it to `s3://concordqa-report-data/workgroup-output/test-resource-YT5DXLKCUhavZKVYQYnd/`.   

Once set, you can perform some queries in the Athena console and the results should be written to the `workgroup-output` folder in the `concordqa-report-data` bucket.  Those queries will appear in the researcher report and can be DLed.

image

### CODE REVIEW

File: researcher-reports/src/components/query-item.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React, { useState, useEffect } from "react";
 import * as AWS from "aws-sdk";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const AmazonS3URI = require("amazon-s3-uri");
```
Comment (scytacki) [inline researcher-reports/src/components/query-item.tsx:4]: Did you try `import AmazonS3URI from "amazon-s3-uri";`

**Review by sfentress — APPROVED** [review]
LGTM

**Review by scytacki — APPROVED** [review]
Looks good to me.

File: researcher-reports/src/components/query-item.tsx (hunk 1 above)
Comment (mklewandowski) [reply researcher-reports/src/components/query-item.tsx]: Ah, I thought I tried that, but I must not have.  I must have been too hung up on the syntax that they presented in their example.  Fixed.

**Pull request merged**